src/compile_torture_tests.py

```diff
@@ -21,24 +21,6 @@
 
 import testing
 
-# TODO: Pass this in from build.py
-INSTALL_DIR = os.path.join(os.getcwd(),
```

Comment:
OK.

How about passing INSTALL_SYSROOT directly from build.py?   When you can make --sysroot and option rather than hardcoding it on line 106?